src/main/java/com/commercetools/sync/commons/utils/SyncUtils.java

```diff
@@ -104,5 +106,17 @@ public static boolean isUuid(@Nonnull final String id) {
     return regexPattern.matcher(id).matches();
   }
 
+  /**
+   * @param key to check if it contains special character(") and format it accordingly.
+   * @return formatted key value.
+   */
+  public static String formatKeyWithEscapeCharacter(final String key) {
```

Comment:
do we really need a separate function for this? Shouldn't it be part of the service class?

Reply:
No more required. I have refactored as part your previous review comment. https://github.com/commercetools/commercetools-sync-java/pull/726/commits/0605901c73598397a4a011b24dfab34cdc594783